editor/editor_node.h

```diff
@@ -646,13 +646,16 @@ class EditorNode : public Node {
 
 	int _get_current_main_editor();
 
-	void _save_docks();
-	void _load_docks();
+	void _save_editor_layout();
+	void _load_editor_layout();
 	void _save_docks_to_config(Ref<ConfigFile> p_layout, const String &p_section);
 	void _load_docks_from_config(Ref<ConfigFile> p_layout, const String &p_section);
 	void _update_dock_slots_visibility(bool p_keep_selected_tabs = false);
 	void _dock_tab_changed(int p_tab);
 
+	void _save_central_editor_layout_to_config(Ref<ConfigFile> p_layout, const String &p_section);
+	void _load_central_editor_layout_from_config(Ref<ConfigFile> p_layout, const String &p_section);
```

Comment:
```suggestion
	void _save_central_editor_layout_to_config(Ref<ConfigFile> p_layout);
	void _load_central_editor_layout_from_config(Ref<ConfigFile> p_layout);
```
I know it's for consistency, but it's pointless if it ever uses only one section.

Reply:
Cleaned these methods up, added a new constant in editor_node.cpp `EDITOR_NODE_CONFIG_SECTION` and replaced all occurences of the string literal "EditorNode" with it.